contracts/OrderRFQMixin.sol

```diff
@@ -56,7 +56,7 @@ abstract contract OrderRFQMixin is EIP712, AmountCalculator, Permitable {
         bytes calldata signature,
         uint256 makingAmount,
         uint256 takingAmount
-    ) external returns(uint256 /* actualMakingAmount */, uint256 /* actualTakingAmount */) {
+    ) external returns(uint256, uint256) {
```

Comment:
I know this was reported but if params were removed from here it should document what's first and what's second argument. It's not clear now when it's just a tuple